docker/openemr/flex-3.10/utilities/devtools

```diff
@@ -12,18 +12,24 @@ if [ "$1" == "build-themes" ]; then
     npm run build
 fi
 
-if [ "$1" == "psr2-report"  ]; then
-    echo "Generating PSR2 code styling error report "
+if [ "$1" == "psr2-report"  ] || [ "$1" == "psr12-report"  ]; then
+    echo "Generating PSR12 code styling error report "
     cd /var/www/localhost/htdocs/openemr
     php -d memory_limit=640M /root/.composer/vendor/squizlabs/php_codesniffer/bin/phpcs -n --extensions=php,inc --standard=ci/phpcs.xml --report=full .
 fi
 
-if [ "$1" == "psr2-fix"  ] || [ "$1" == "clean-sweep" ]; then
-    echo "Fixing PSR2 code styling errors "
+if [ "$1" == "psr2-fix"  ] ||  [ "$1" == "psr12-fix"  ] ||[ "$1" == "clean-sweep" ]; then
```

Comment:
hi @prondubuisi 
bash can be temperamental with spacing on conditionals, so rec having a space above between the `||` and `[`

Reply:
Fixed @bradymiller thanks. I need those Eagle eyes of yours!